service/history/workflow/mutable_state_rebuilder_test.go

```diff
@@ -157,8 +157,8 @@ func (s *stateBuilderSuite) mockUpdateVersion(events ...*historypb.HistoryEvent)
 	s.mockMutableState.EXPECT().SetHistoryBuilder(NewImmutableHistoryBuilder(events))
 }
 
-func (s *stateBuilderSuite) toHistory(events ...*historypb.HistoryEvent) []*historypb.HistoryEvent {
-	return events
+func (s *stateBuilderSuite) toHistory(eventss ...*historypb.HistoryEvent) [][]*historypb.HistoryEvent {
```

Comment:
I felt the mixed use of `...` and `[]` weird and not really consistent. is it some sort of convention on when to use `...` and when to use `[]`?

Reply:
1. this is test function
2. ... vs [] actually depends on the business logic context: here, the logic need to apply []*historypb.HistoryEvent as a transaction, and we are trying to combine multiple transactions into save IO